client/verta/verta/_internal_utils/model_dependencies.py

```diff
@@ -73,7 +74,11 @@ def class_module_names(model_class: Type[VertaModelBase]) -> Set[str]:
     return modules_found
 
 
-# TODO VRD-682 convert module names to pkg names here
-def package_names(module_names: Set[str]) -> Set[str]:
-    """Return a set of all packages detected in the provided set of base module names."""
-    return module_names
+def package_names(module_names: Set[str]) -> Dict[str, List[str]]:
+    """Return a dictionary where the key is the name of the import module
+    and the value is the list of possible distribution packages for each
+    3rd-party import module in the provided set that can be found in the
+    locally installed package_distributions.
+    """
+    pkg_dist = packages_distributions()
+    return {m: pkg_dist.get(m) for m in module_names if pkg_dist.get(m)}
```

Comment:
I know you mentioned covering this in a follow-up PR, but I'd strongly recommend putting at least a `# TODO` here about cases where `pkg_dist.get(m)` returns `None`. It's a sneaky case that's hard to catch when just looking at this code, and I don't want it to be missed forever (if, say, we end up not getting around to the follow-up).

Reply:
Done